src/bitmessageqt/addressvalidator.py

```diff
@@ -60,25 +80,30 @@ def checkQueue(self):
                 if gotOne:
                     break
                 else:
-                    return
+                    return None
             else:
                 gotOne = True
 
-        if len(addressGeneratorReturnValue) == 0:
+        if not addressGeneratorReturnValue:
             self.setError(_translate("AddressValidator", "Address already present as one of your identities."))
             return (QtGui.QValidator.Intermediate, 0)
         if addressGeneratorReturnValue[0] == 'chan name does not match address':
-            self.setError(_translate("AddressValidator", "Although the Bitmessage address you entered was valid, it doesn\'t match the chan name."))
+            self.setError(
+                _translate(
+                    "AddressValidator",
+                    "Although the Bitmessage address you \
```

Comment:
Please no. Splitting strings into multiple lines should be like this:

```
    " aaaaa "
    " bbbb "
````

And there shouldn't be `\'` inside `""`.